app/controllers/projects_controller.rb

```diff
@@ -75,7 +75,7 @@ def new_team_for_project
   end
 
   def add_team_to_project
-    team = Team.find_by_id params[:project][:team]
+    team = current_company.teams.find_by_id params[:project][:team]
```

Comment:
```suggestion
    team = @current_company.teams.find_by_id params[:project][:team]
```
Use find_by instead of find_by_*

Reply:
current_company -> @current_company 
Verify this change throughout